### PR TITLE
Styling bootstrap

### PR DESCRIPTION
**Pull request by AnthonyTC89**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently under review

File: css/style.css [hunk 1]
```diff
@@ -1,7 +1,242 @@
...
-    margin: 0 1rem;
-}
+
+/* Extra small devices (phones, less than 768px) */
 .header{
-    background-color: #F72210;
-    color: #FFF;
-}
\ No newline at end of file
+  justify-content: start;
```
Comment (bolah2009) [inline css/style.css]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

File: css/style.css [hunk 2]
```diff
@@ -1,7 +1,242 @@
...
+  left: 4rem;
+  top: 2rem;
+}
+
+.subscribe, .newsletter-part{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
```
Comment (bolah2009) [inline css/style.css]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

File: css/style.css [hunk 3]
```diff
@@ -1,7 +1,242 @@
...
+    justify-content: flex-end;
+    align-items: center;
+    text-align: start;
+    padding-top: 0; 
+    margin-left: 0;
+  }
+  .newsletter-part{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
```
Comment (bolah2009) [inline css/style.css]: `SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Awesome Job [Anthony](../../) :clap: 

Awesome job so far, the [clone](https://raw.githack.com/AnthonyTC89/newsweek-clone/styling-bootstrap/index.html) looks good to me and very responsive.. :clap:

However, use more of bootstrap as pointed out in [this comment](https://github.com/AnthonyTC89/newsweek-clone/pull/3#discussion_r311325279) and all other occurrences and make the website look more like the original one as shown below (currently there are too many spaces by the sides of the website)


|Current|Preffered|
| :-----: | :------: |
| newsweek current space  | newsweek preffered space |

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

File: css/style.css (hunk 1 above)
Comment (AnthonyTC89) [reply css/style.css]: done

File: css/style.css (hunk 2 above)
Comment (AnthonyTC89) [reply css/style.css]: done

File: css/style.css (hunk 3 above)
Comment (AnthonyTC89) [reply css/style.css]: done

**Comment by samgaco** [comment]
### Hi @AnthonyTC89 :hand: 

### Awesome project overall, it's approved. Congratulations :clap: :tada: 

**[OPTIONAL]** _You can still work to make the styling look closer to the one in the original website as suggested by @bolah2009 . A tool I like to use to help me out with that is[ Eye Dropper](https://chrome.google.com/webstore/detail/eye-dropper/hmdcmlfkchdmnmnmheododdhjedfccka)._

### Anyways the project looks truly good, so keep the good work moving on. Regards! :muscle:

**Pull request merged**